addons/account_edi_proxy_client/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
+"access_account_edi_proxy_user","access_account_edi_proxy_user","model_account_edi_proxy_client_user","base.group_system",1,1,1,0
```

Comment:
we never delete edi proxy users?

Reply:
Since some data is encrypted on the proxy and only this user has the private key, we never delete them but rather set them to `active=False` in case something goes wrong and we need to recover the private key.